beacon_node/builder_client/src/lib.rs

```diff
@@ -57,14 +62,20 @@ pub struct BuilderHttpClient {
     server: SensitiveUrl,
     timeouts: Timeouts,
     user_agent: String,
-    ssz_enabled: Arc<AtomicBool>,
+    /// Only use json for all requests/responses types.
+    disable_ssz: bool,
+    /// Indicates that the `get_header`` response had content-type ssz
+    /// so we can set content-type header to ssz to make the `submit_blinded_blocks`
+    /// request.
+    ssz_used: Arc<AtomicBool>,
```

Comment:
This might be a better name?

```suggestion
    ssz_available: Arc<AtomicBool>,
```